[intel-distill] 审核工作流无法正常工作：review-state.ts 缺失，update-state.ts 不支持审核参数

## 问题描述

`/intel-distill` 命令的审核工作流（`--review approve/reject`）无法正常工作，导致待审核队列无法被正确处理。

## 根因分析

### 1. `update-state.ts` 参数缺失

脚本未实现文档描述的审核参数：

| 参数 | 文档描述 | 实际实现 |
|------|---------|---------|
| `--pending-id` | ✅ 支持 | ❌ 未解析 |
| `--action approve/reject` | ✅ 支持 | ❌ 未解析 |

代码位置：`scripts/preprocess/update-state.ts` 第 349-365 行
```typescript
// 只解析 --state, --results, --output
// 未解析 --pending-id, --action
```

### 2. `review.pending` 无法移除

脚本第 327-336 行只实现"添加 pending"，未实现"移除 pending"：
```typescript
if (result.has_strategic_value === null) {
  state.review.pending.push({...});  // 只会 push，不会 remove
}
```

### 3. `review-state.ts` 脚本缺失

skill 文档隐含引用 `review-state.ts`，但该脚本不存在：
```bash
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '.../review-state.ts'
```

### 4. 输出误导

脚本返回"成功"但 pending 数组未变化：
```
✅ State updated successfully
   Pending review: 1   ← 数量未变化
```

## 影响范围

| 问题 | 影响 | 严重度 |
|------|------|--------|
| `--pending-id` 未解析 | 审核命令无效 | 🔴 高 |
| `--action` 未解析 | approve/reject 无效 | 🔴 高 |
| pending 无法移除 | 队列只增不减 | 🔴 高 |
| review-state.ts 缺失 | 调用报错 | 🔴 高 |
| 输出误导 | 用户误判状态 | 🟡 中 |

## 解决建议

### 方案 A：创建独立审核脚本（推荐）

创建 `scripts/preprocess/review-state.ts`：

```typescript
interface ReviewArgs {
  pendingId: string;
  action: 'approve' | 'reject';
  reason: string;
  outputDir: string;
}

function executeReview(args: ReviewArgs): void {
  // 1. 加载 state.json
  // 2. 从 review.pending 中查找并移除 pendingId
  // 3. 更新 processed[key].review_status
  // 4. 更新 stats.review.pending/approved/rejected
  // 5. 保存 state.json
}
```

### 方案 B：扩展 update-state.ts

在现有脚本添加审核分支：
```typescript
if (pendingId && action) {
  executeReview(state, pendingId, action, reason);
}
```

## 相关文件

- `commands/intel-distill.md` - skill 文档
- `scripts/preprocess/update-state.ts` - 状态更新脚本
- `scripts/preprocess/` - 缺少 review-state.ts

## 验证步骤

```bash
# 1. 运行情报提取，产生待审核项
/intel-distill

# 2. 尝试审核
/intel-distill --review list
/intel-distill --review approve pending-xxx --reason "测试"

# 3. 检查 state.json
# 预期：pending 数组减少，review_status 更新
# 实际：pending 未变化，脚本返回"成功"
```

--- a/plugins/market-radar/commands/intel-distill.md
+++ b/plugins/market-radar/commands/intel-distill.md
@@ -1,7 +1,7 @@
 ---
 name: intel-distill
 description: Extract strategic intelligence from source documents and generate intelligence cards
-argument-hint: "[--source <目录>] [--output <目录>] [--review <list|approve|reject> [pending_id] [--reason <原因>]] [--report <daily|weekly|monthly> [--date <日期>|周期]]"
+argument-hint: "[--source <目录>] [--output <目录>] [--review <list|approve|reject> [--pending-id <id>] [--reason <原因>]] [--report <daily|weekly|monthly> [--date <日期>|周期]]"
 allowed-tools: Read, Write, Grep, Glob, Bash, Agent
 ---
 
@@ -20,6 +20,7 @@ allowed-tools: Read, Write, Grep, Glob, Bash, Agent
 | `--source <dir>` | 否 | 待扫描的源文件目录（默认：./inbox） |
 | `--output <dir>` | 否 | 情报卡片输出目录（默认：./intelligence） |
 | `--review <action>` | 否 | 审核操作：`list`/`approve`/`reject` |
+| `--pending-id <id>` | 条件 | 待审核项 ID（approve/reject 时必填） |
 | `--reason <text>` | 条件 | 审核原因（approve/reject 时推荐） |
 | `--report <type>` | 否 | 生成情报简报：`daily`/`weekly`/`monthly` |
 | `--date <YYYY-MM-DD>` | 否 | 日报日期（仅 daily 模式，默认当天） |
@@ -47,10 +48,10 @@ allowed-tools: Read, Write, Grep, Glob, Bash, Agent
 /intel-distill --review list
 
 # 批准待审核项
-/intel-distill --review approve pending-threat-20260313-001 --reason "情报准确"
+/intel-distill --review approve --pending-id pending-threat-20260313-001 --reason "情报准确"
 
 # 拒绝待审核项
-/intel-distill --review reject pending-emerging-20260313-002 --reason "信息来源不可靠"
+/intel-distill --review reject --pending-id pending-emerging-20260313-002 --reason "信息来源不可靠"
 
 # === 报告模式 ===
 
@@ -276,7 +277,7 @@ intel-pull 采集 → inbox/*.md frontmatter → 预处理转换 → converted/*
 - source_dir = --source 参数或 ./inbox
 - output_dir = --output 参数或 ./intelligence
 - review_action = --review 参数值（list/approve/reject 或无）
-- review_target = 审核目标 pending_id
+- pending_id = --pending-id 参数值（approve/reject 时需要）
 - review_reason = --reason 参数值
 - report_type = --report 参数值（weekly/monthly 或无）
 - period_param = 周期参数
@@ -708,11 +709,11 @@ Agent 直接更新转换文件 frontmatter 和 pending.json：
    添加时间: 2026-03-13 10:05:00
 
 💡 操作:
-   /intel-distill --review approve <pending_id> --reason "原因"
-   /intel-distill --review reject <pending_id> --reason "原因"
+   /intel-distill --review approve --pending-id <pending_id> --reason "原因"
+   /intel-distill --review reject --pending-id <pending_id> --reason "原因"
 ```
 
-### 步骤 A2：批准审核（`--review approve <pending_id> --reason`）
+### 步骤 A2：批准审核（`--review approve --pending-id <id> --reason`）
 
 1. **查找待审核项**
    - 从 `pending.json` 的 `review.pending` 中查找 `pending_id`
@@ -752,7 +753,7 @@ Agent 直接更新转换文件 frontmatter 和 pending.json：
    • 批准原因已记录（正样本反馈）
 ```
 
-### 步骤 A3：拒绝审核（`--review reject <pending_id> --reason`）
+### 步骤 A3：拒绝审核（`--review reject --pending-id <id> --reason`）
 
 1. **查找待审核项**
    - 从 `pending.json` 的 `review.pending` 中查找 `pending_id`

